internal/store/test_common.go

```diff
@@ -0,0 +1,73 @@
+package store
```

Comment:
these get compiled into the binary can we put these helpers in test/util/util.go along with any other test helpers.

Reply:
That was a pain to change without cyclic imports, but did it :)